FileManager.h

```diff
@@ -0,0 +1,13 @@
+
+//sample setup for FileManager header file
+
+using namespace std;
```

Comment:
It's good practice to explicitly use std:: throughout ones program. I suggest you do so.